ground/gcs/src/plugins/config/configoutputwidget.cpp

```diff
@@ -413,7 +413,17 @@ void ConfigOutputWidget::startESCCalibration()
             if (!channelsToCalibrate[i])
                 continue;
 
-            actuatorCommandFields.Channel[i] = actuatorSettingsData.ChannelMin[i];
+            if (actuatorSettingsData.ChannelMax[i] >
+                    (actuatorSettingsData.ChannelMin[i] + 40)) {
+                // If it is reasonable to do it, calibrate the minimum of the range
+                // to be a little off what we send at idle.  Blheli takes the value
+                // we give verbatim as the minimum, and small amounts of jitter
+                // can cause the motor to tick over.
+                actuatorCommandFields.Channel[i] =
+                    actuatorSettingsData.ChannelMin[i] + 3;
+            } else {
+                actuatorCommandFields.Channel[i] = actuatorSettingsData.ChannelMin[i];
+            }
```

Comment:
What about this? Handles the case where min and max are inverted, and any pulse length. I chose / 40 to get close to the 3 us you were presumably intending to add to Oneshot125. Limit to max adjustment of 5 so oldschool PWM doesn't go crazy.
https://ideone.com/2m38QK

Really it would be better to fix this in BLHeli since it's their bug...